src/azure-cli-core/azure/cli/core/commands/client_factory.py

```diff
@@ -110,12 +110,21 @@ def configure_common_settings(cli_ctx, client):
     client.config.generate_client_request_id = 'x-ms-client-request-id' not in cli_ctx.data['headers']
 
 
-def configure_common_settings_track2(cli_ctx):
+def _prepare_client_kwargs_track2(cli_ctx):
+    """Prepare kwargs for Track 2 SDK client."""
     client_kwargs = {}
 
+    # Prepare connection_verify to change SSL verification behavior, used by ConnectionConfiguration
     client_kwargs.update(_debug.change_ssl_cert_verification_track2())
 
+    # Enable NetworkTraceLoggingPolicy which logs all headers (except Authorization) without being redacted
     client_kwargs['logging_enable'] = True
+
+    # Disable ARMHttpLoggingPolicy which logs only allowed headers
+    from azure.core.pipeline.policies import SansIOHTTPPolicy
+    client_kwargs['http_logging_policy'] = SansIOHTTPPolicy()
```

Comment:
@qwordy, This will remove the duplicated network trace.

```log
azure.core.pipeline.policies._universal : Request URL: 'https://management.azure.com/subscriptions/0b1f6471-1bf0-4dda-aec3-cb9272f09590?api-version=2016-06-01'
azure.core.pipeline.policies._universal : Request method: 'GET'
azure.core.pipeline.policies._universal : Request headers:
azure.core.pipeline.policies._universal :     'Accept': 'application/json'
...
azure.core.pipeline.policies.http_logging_policy : Request URL: 'https://management.azure.com/subscriptions/0b1f6471-1bf0-4dda-aec3-cb9272f09590?api-version=REDACTED'
azure.core.pipeline.policies.http_logging_policy : Request method: 'GET'
azure.core.pipeline.policies.http_logging_policy : Request headers:
azure.core.pipeline.policies.http_logging_policy :     'Accept': 'application/json'
...
azure.core.pipeline.policies.http_logging_policy : Response status: 200
azure.core.pipeline.policies.http_logging_policy : Response headers:
azure.core.pipeline.policies.http_logging_policy :     'Cache-Control': 'no-cache'
...
azure.core.pipeline.policies._universal : Response status: 200
azure.core.pipeline.policies._universal : Response headers:
azure.core.pipeline.policies._universal :     'Cache-Control': 'no-cache'
...
```

Logs from `azure.core.pipeline.policies.http_logging_policy` will now be disabled.